bigflow/build/operate.py

```diff
@@ -200,9 +276,15 @@ def build_project(
     project_spec: BigflowProjectSpec,
     start_time: str,
     workflow_id: typing.Optional[str] = None,
+    export_image_tar: bool | None = None,
```

Comment:
General comment - there is inconsistency in optional type hints. In some places the old form (Optional[T]) is used, in some the new (Optional | T). Look at the previous line. 
Shouldn't we choose one option?

Reply:
Especially in the context of one function.

Reply:
Sure, it's good to have consistent type-hints in a single declaration, I've just missed that (fixed).

In a wider sense - previously we used only "proper" annotations for "python 3.7" (`Option`, `Union` etc).
Now a "new-style" annotation from newer version of python are used, but old code remains the same, so we have some "mix". IMO there is no need to change hints immediately, but for new code I prefer "new style" where it is possible.

Also note then new-style `X | None`, `list[X]` are working only because of `from __future__ import annotations` declaration, which instructs python to keep annotations as a strings. So it is possible to write any "not working" expression if it may be parsed into correct python AST. However tools (at least mypy & pyright/pylance) know new ways of declaring type-hints and use them correctly.